analytics/views.py

```diff
@@ -1139,7 +1139,7 @@ def support(request: HttpRequest) -> HttpResponse:
         scrub_realm = request.POST.get("scrub_realm", None)
         if scrub_realm is not None:
             if scrub_realm == "scrub_realm":
-                do_scrub_realm(realm)
+                do_scrub_realm(realm, request.user)
```

Comment:
Yeah, this is the right approach to use for anything from `support`.  I'd prefer to pass this as kwargs BTW, `acting_user=request.user`.